### PR TITLE
819-1: operation cost: wails front

### PR DESCRIPTION
**Pull request by Thykof**
This PR improve the sign prompt and allow the user to modify the fees, but the modification is not taken into account (see following PR branch 819-2)

### CODE REVIEW

File: wails-frontend/src/pages/PasswordPromptHandler/OperationCost.tsx [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+                    disabled
+                    value={formatStandard(coins)}
+                  />
+                </div>
+              )}
+              <div className="flex flex-row justify-end gap-1">
+                {isEditing ? (
+                  <>
+                    <button
+                      className="flex flex-row hover:cursor-pointer"
```
Comment (pivilartisant) [inline wails-frontend/src/pages/PasswordPromptHandler/OperationCost.tsx:139]: tailwind tip: when using the tag "flex" attr "flex-row" is default, so you can safely remove  "flex-row"

File: wails-frontend/src/pages/PasswordPromptHandler/Sign.tsx [hunk 2]
```diff
@@ -105,14 +119,47 @@ export function Sign() {
           {(() => {
             switch (signData.OperationType) {
               case OPER_CALL_SC:
-                return <CallSc {...signData} />;
+                return (
```
Comment (pivilartisant) [inline wails-frontend/src/pages/PasswordPromptHandler/Sign.tsx:122]: Concerning the fees being disabled during, i believe that passing isEditing as a param to OperationCost would make sense... tdb in small follow up pr

**Review by pivilartisant — APPROVED** [review]
I left 3 comments, nothing major but otherwise: LGTM!

File: wails-frontend/src/pages/PasswordPromptHandler/OperationCost.tsx (hunk 1 above)
Comment (Thykof) [reply wails-frontend/src/pages/PasswordPromptHandler/OperationCost.tsx:139]: ok but there are many other flex flex-row in the code, maybe another PR to simplify all this

Comment (pivilartisant) [reply wails-frontend/src/pages/PasswordPromptHandler/OperationCost.tsx:139]: Totally

**Review by thomas-senechal — APPROVED** [review]
Backend looks good to me

**Pull request merged**